base/doc/ltnews40.tex

```diff
@@ -348,24 +474,25 @@ \subsection{Improved error raised by empty hook}
 
 \subsection{Provide counter representations for link targets}
 
-To create unique target names for links the package 
-\pkg{hyperref} uses a special counter representation
-\verb+\theH+\meta{counter}. To ensure that this 
-counter representation exists, \pkg{hyperref} redefined the 
-commands \verb+\@definecounter+, \verb+\@addtoreset+ 
-and \verb+\refstepcounter+. This counter representation is also
-needed for the Tagged PDF project and and so these augmented command definitions 
-have now been incorporated
-into the kernel. 
-Thus from now on every \verb+\newcounter{+\meta{counter}\verb+}+ will not
-only define \verb+\the+\meta{counter} but also \verb+\theH+\meta{counter}.
+To create unique target names for links the package \pkg{hyperref}
+uses a special counter representation \verb+\theH+\meta{counter}. To
+ensure that this counter representation exists, \pkg{hyperref}
+redefined the commands \verb+\@definecounter+, \verb+\@addtoreset+ and
+\verb+\refstepcounter+. This counter representation is also needed for
+the Tagged PDF project and so these augmented command definitions
+have now been incorporated into the kernel.  Thus from now on every
+\verb+\newcounter{+\meta{counter}\verb+}+ will not only define
+\verb+\the+\meta{counter} but also \verb+\theH+\meta{counter}.
 
 \subsection{Extending \cs{refstepcounter}}
 
-The package \pkg{hyperref} redefines since many years \verb+\refstepcounter+ and
-adds code that creates link targets. The kernel definition has now been extended
-with socket interfaces that will allow \pkg{hyperref} to avoid the redefinitions. The new interfaces are also used by the Tagged PDF code that needs target names to resolve
-references between structures.
+For many years, the package \pkg{hyperref} has redefined
+\verb+\refstepcounter+ and adds code that creates link targets. The
```

Comment:
You jumped tenses here.
```suggestion
\verb+\refstepcounter+ and added code that creates link targets. The
```

Also if my grammar lessons from the back of my memory serve me well, correct tense here would be past perfect progressive, so this sentence should use

```
For many years, the package \pkg{hyperref} has been redefining
\verb+\refstepcounter+ and adding code that creates link targets.
```